packages/synapse-react-client/src/components/SubscriptionPage/SubscriptionPage.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useId, useState } from 'react'
+import { useGetAllSubscriptions } from '../../synapse-queries/subscription/useSubscription'
+import {
+  SortByType as SubscriptionSortField,
+  SortDirection,
+  SubscriptionObjectType,
+} from '@sage-bionetworks/synapse-types'
+import SubscriptionItem from './SubscriptionItem'
+import { Box, Button, Stack, Tab, Tabs } from '@mui/material'
+import DropdownSelect from '../DropdownSelect'
+
+type SortOption = { value: SortDirection; label: string }
+
+const sortOptions: SortOption[] = [
+  { value: 'ASC', label: 'Date Posted' },
+  { value: 'DESC', label: 'Most Recent' },
+]
+
+export default function SubscriptionPage() {
+  const [objectType, setObjectType] = useState<SubscriptionObjectType>(
+    SubscriptionObjectType.FORUM,
+  )
+  const [sortDirection, setSortDirection] = useState<SortOption>(sortOptions[0])
+
+  const instanceSpecificQueryKey = useId()
+
+  const currentFilter = {
+    objectType: objectType,
+    sortBy: SubscriptionSortField.CREATED_ON,
+    sortDirection: sortDirection.value,
+  }
+
+  const { data, hasNextPage, isFetchingNextPage, fetchNextPage } =
+    useGetAllSubscriptions(
+      currentFilter,
+      // Since the user may edit their subscriptions on this page, set staleTime to infinity to prevent re-fetching while editing
+      { staleTime: Infinity },
+      // Override the query key with a unique ID, because we don't want changes to invalidate this list while it's being used
+      [`subscriptionPageFetch_${instanceSpecificQueryKey}`, currentFilter],
```

Comment:
I think I understand.  So the topic does not disappear from the UI when the user clicks unsubscribe, right?

Reply:
Exactly!